pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -95,9 +95,9 @@ class PagerState(
     internal var pageSize by mutableStateOf(0)
 
     /**
-     * The page position, as a float value between `0 until pageSize`
+     * The current scroll position, as a float value between `0 until pageSize`
      */
-    private val globalPosition: Float
+    val scrollPosition: Float
```

Comment:
`scrollPosition` seems to imply a px value to me but Im struggling to come up with better alternatives. `currentPosition`? `absolutePosition`?

Reply:
`scrollPosition` isn't great, but I can't think of anything better either. `scrollPosition` works ok, it's just the assumption of the unit. Considering now other part of the API uses `px`, I think the API name is ok

Reply:
SG.